src/test/groovy/com/github/vkorobkov/jfixtures/loader/RowTest.groovy

```diff
@@ -94,4 +94,43 @@ class RowTest extends Specification {
         expect:
         EqualsVerifier.forClass(Row).verify()
     }
+
+    def "#columns returns row from key/value pairs"() {
```

Comment:
Maybe since we have overloaded `columns` method it makes sense to reflect that fact in tests:
```java
def "#columns(Object...) returns row from key/value pairs"()
```